assets/prometheus-operator/deployment.yaml

```diff
@@ -43,13 +44,18 @@ spec:
             memory: 60Mi
         securityContext: {}
         terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - mountPath: /etc/tls/private
+          name: prometheus-operator-tls
+          readOnly: false
       - args:
         - --logtostderr
         - --secure-listen-address=:8443
         - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
```

Comment:
Should this not be --web.tls-cipher-suites?

Reply:
Hmm, I think you are right, I'm not sure why this wasn't caught by the tests.

Reply:
Ok, that one is used by the kube-rbac-proxy and not by prometheus-operator, so it should be fine.  This PR currently doesn't set the cipher-suites for prometheus-operator, so it will just use the default ones providied by the golang API.  I guess this should be updated to match the ones used by kube-rbac-proxy?  Or is there another standard set we should be using for openshift?

Reply:
I'll update the jsonnet to pull from the same ciphersuites field as the other assets.

Reply:
I've added the ciphersuites to the prometheus-operator jsonnet config, so it should match the other deployments now.